js/main.js

```diff
@@ -1,33 +1,156 @@
-
-//Функция, возвращающая случайное число с плавающей точкой из переданного диапазона включительно.
-const getRandomWithPoint = (min, max, point) => {
-  if (max === undefined || point === undefined || min === undefined) {
-    return 'Введите все аргументы функции';
+const getRandomPositiveFloat = (a, b, digits = 1) => {
+  const lower = Math.min(Math.abs(a), Math.abs(b));
+  const upper = Math.max(Math.abs(a), Math.abs(b));
+  const result = Math.random() * (upper - lower) + lower;
+  return +result.toFixed(digits);
+};
+const getRandomArray = (array) => {
+  const maxLength = array.length;
+  const lengthOfArray = getRandomPositiveInteger(1, maxLength);
+  const newArray = [];
+  while (newArray.length < lengthOfArray) {
+    const indexOfEl = getRandomPositiveInteger(0, maxLength - 1);
+    const el = array[indexOfEl];
+    if (!newArray.includes(el)) {
+      newArray.push(el);
+    }
   }
-  if (max === min) {
-    return 'Диапазоном не может быть одно число';
+  return newArray;
+};
+const arrayIdAvatarCreate = () => {
+  const array = [];
+  for (let i = 0; i < SIMILAR_ADVERTISEMENTS_COUNT; i++) {
+    array[i] = i + 1;
   }
-  if (min > max) {
-    return 'Введите корректный диапазон "от...до"';
+  return array;
+};
+const arrayId = arrayIdAvatarCreate();
+const minPrice = 5000;
+const maxPrice = 100000;
+const maxRoomCount = 100;
+const maxGuestsCount = 10;
+const latitudeFrom = 35.65000;
+const latitudeTo = 35.70000;
+const longitudeFrom = 139.70000;
+const longitudeTo = 139.80000;
+const headline = [
+  'Квартирка для котов лентяев',
+  'Дворец для состоятельных мурчал',
+  'Сдаю гараж с мышами',
+  'Дом для многодетных котов',
+  'Валерианин дом',
+];
+const DESCRIPTION_OF_HOUSING = [
+  'Ночлежка для ночных котобродяг',
+  'Бесплатный вискас, молоко, когтеточка, массаж и мурмяу',
+  'Отель без хозяев, без котодетей, без котообязательств',
+  'Место для котов без хвостов и совести',
+  'Место чтоб залечь на дно, если натворил кое-каких дел',
+  'Аппартаменты для котобоссов и котобоссок',
+];
+const PHOTO_OF_HOUSING = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+const TYPE_HOUSING = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+const FEATURES_OF_HOUSING = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner'
+];
+const CHECK_IN_TIME = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+const CHECK_OUT_TIME = CHECK_IN_TIME.slice();
+const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
+const createAvatarAddress = () => {
```

Comment:
```suggestion
const createAvatarSrc = () => {
```

как называется атрибут в img, в котором пишем урл картинки? Разве адрес?